portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -4,7 +4,6 @@
   <sessions-enabled>true</sessions-enabled>
   <runtime>java8</runtime>
   <static-files>
-    <!-- prevent unwanted caching when accessing via the web preview server -->
```

Comment:
nit: We generally try to avoid making changes which are not related to our PR

("nit" means that the reviewer thinks this is odd and something you might have forgot while coding. Not actually a comment on the code or the design.)